Bottom: 98794926443a356eddad3b2790d57a5e73abfbe1
Top:    326b0890e52ba05ea31402a5c789708a80cf3f24
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 17:20:14 -0500

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 4bc02cc..8927fa4 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -997,10 +997,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
